packages/yoroi-extension/app/styles/themes/revamp/base-palettes/dark-palette.js

```diff
@@ -0,0 +1,58 @@
+export const darkPalette = {
```

Comment:
This files are added in this PR https://github.com/Emurgo/yoroi-frontend/pull/3493, shouldn't be here

Reply:
@nistadev the PR - https://github.com/Emurgo/yoroi-frontend/pull/3493 is stuck for now and can't block all future development and theme progress because of it - as a solution had to bring the new tokens and work with them on the new feature as we don't know yet when the dark theme PR will be ready (I have made the same for the external Vendors).  cc @vsubhuman. 
  After this PR will be merged will have this:   
- new design tokens and palette `ds` will be available to work with it in dev  😎 
- will need to merge https://github.com/Emurgo/yoroi-frontend/pull/3493  with development and resolve there the conflicts and refactor maybe a bit
- I will take the task of the dark theme PR next as I understand from @vsubhuman that is a priority